### PR TITLE
[#172172927] update RemindEmailValidationOverlay ui

### PR DESCRIPTION
**Pull request by valentinacala**
**Short description:**
This pr updates the ui of the content displayed in the RemindEmailValidationOverlay:

**WHEN THE MAIL SHOULD BE VALIDATED - DURING ONBOARDING:**
image

**WHEN THE MAIL SHOULD BE VALIDATED - AFTER ONBOARDING:**
image

image

**WHEN THE EMAIL IS VALIDATED AND THE FOCUS IS ON THE OVERLAY:**
image

### CODE REVIEW

File: ts/components/RemindEmailValidationOverlay.tsx [hunk 1]
```diff
@@ -303,39 +307,53 @@ class RemindEmailValidationOverlay extends React.PureComponent<Props, State> {
...
-          <React.Fragment>
-            <Image style={styles.imageChecked} source={image} />
-            <View spacer={true} extralarge={true} />
-          </React.Fragment>
-          <H2 style={isOnboardingCompleted ? styles.emailTitle : undefined}>
+        <Content bounces={false}>
+          <View spacer={true} extralarge={true} />
+          <IconFont
+            name={icon}
+            size={84}
```
Comment (Undermaken) [inline ts/components/RemindEmailValidationOverlay.tsx:330]: could it be a const?

File: ts/components/RemindEmailValidationOverlay.tsx [hunk 2]
```diff
@@ -303,39 +307,53 @@ class RemindEmailValidationOverlay extends React.PureComponent<Props, State> {
...
+          <Text alignCenter={true} bold={true}>
             {title}
-          </H2>
-          <View spacer={true} />
+          </Text>
           {!this.state.emailHasBeenValidate ? (
-            <Markdown onLoadEnd={this.handleOnContentLoadEnd}>
+            <Markdown
+              onLoadEnd={this.handleOnContentLoadEnd}
+              cssStyle={"body { text-align: center;}"}
```
Comment (Undermaken) [inline ts/components/RemindEmailValidationOverlay.tsx:341]: could it be a const?

Comment (valentinacala) [reply ts/components/RemindEmailValidationOverlay.tsx]: updated! https://github.com/pagopa/io-app/pull/1658/commits/d8892ad036d58806a7f930d89aacb8b451655d2b

File: ts/components/RemindEmailValidationOverlay.tsx (hunk 1 above)
Comment (valentinacala) [reply ts/components/RemindEmailValidationOverlay.tsx]: updated! https://github.com/pagopa/io-app/pull/1658/commits/d8892ad036d58806a7f930d89aacb8b451655d2b

**Pull request merged**